.github/workflows/beam_LoadTests_Go_Combine_Flink_Batch.yml

```diff
@@ -0,0 +1,133 @@
+    timeout-minutes: 720
+    name: ${{ matrix.job_name }} (${{ matrix.job_phrase }})
+    strategy:
+      matrix:
+        job_name: ["beam_LoadTests_Go_Combine_Flink_Batch"]
+        job_phrase: ["Run Load Tests Go Combine Flink Batch"]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup repository
+        uses: ./.github/actions/setup-action
+        with:
+          comment_phrase: ${{ matrix.job_phrase }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          github_job: ${{ matrix.job_name }} (${{ matrix.job_phrase }})
+      - name: Prepare test arguments
+        uses: ./.github/actions/test-arguments-action
+        with:
+          test-type: load
+          test-language: go
+          argument-file-paths: |
+            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/config_Combine_Go_Batch_Flink_10b.txt
+            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/config_Combine_Go_Batch_Flink_Fanout_4.txt
+            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/config_Combine_Go_Batch_Flink_Fanout_8.txt
+          arguments: |
+            --job_name=load-tests-go-flink-batch-combine-$(date '+%m%d%H%M%S' --utc)
+      - name: Start Flink with parallelism 5
+        env:
+          FLINK_NUM_WORKERS: 5
+        run: |
+          cd ./.test-infra/dataproc; ./flink_cluster.sh create
+      - name: run Combine Flink Batch Go Load Test 1 (10b records)
+        timeout-minutes: 240
+        uses: ./.github/actions/gradle-command-self-hosted-action
+        with:
+          gradle-command: :sdks:go:test:load:run
+          arguments: |
+            -PloadTest.mainClass=combine \
+            -Prunner=FlinkRunner \
+            '-PloadTest.args=${{ env.beam_LoadTests_Go_Combine_Flink_Batch_test_arguments_1 }}' \
+      - name: Restart Flink with parallelism 16
+        env:
+          FLINK_NUM_WORKERS: 16
+        run: |
+          cd ./.test-infra/dataproc; ./flink_cluster.sh restart
+      - name: run Combine Flink Batch Go Load Test 2 (fanout 4)
+        timeout-minutes: 240
+        uses: ./.github/actions/gradle-command-self-hosted-action
+        with:
+          gradle-command: :sdks:go:test:load:run
+          arguments: |
+            -PloadTest.mainClass=combine \
+            -Prunner=FlinkRunner \
+            '-PloadTest.args=${{ env.beam_LoadTests_Go_Combine_Flink_Batch_test_arguments_2 }}' \
+      - name: run Combine Flink Batch Go Load Test 3 (fanout 8)
+        timeout-minutes: 240
+        uses: ./.github/actions/gradle-command-self-hosted-action
+        with:
+          gradle-command: :sdks:go:test:load:run
+          arguments: |
+            -PloadTest.mainClass=combine \
+            -Prunner=FlinkRunner \
+            '-PloadTest.args=${{ env.beam_LoadTests_Go_Combine_Flink_Batch_test_arguments_3 }}'
+      - name: Teardown Flink
+        if: always()
+        run: |
+          ./.test-infra/dataproc/flink_cluster.sh delete
```

Comment:
let's add the `${{ github.workspace }}` at the begining in order to have an absolute path

Reply:
Done